### PR TITLE
Refs 9207

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1 +1,5 @@
-2014-07-27 Release 0.1.0
```
Comment (stbenjam) [inline CHANGELOG]: I'd rename this to .md if it'll be in markdown

**Comment by ekohl** [comment]
You can also drop ```foreman_oauth_key``` from ```params.pp```. I'd also simplify the comment above it because I think it's no longer relevant.

**Comment by ehelms** [comment]
@stbenjam could you look at https://github.com/Katello/puppet-katello/pull/50 as I'd like to get that in before this.

**Comment by ekohl** [comment]
@ehelms I think this needs a rebase.

**Comment by ehelms** [comment]
@ekohl Will do. I was thinking of doing the modulesync part first, and then re-basing and revisiting this.

**Comment by ehelms** [comment]
@ekohl @stbenjam Updated and rebased this post modulesync updates.

**Comment by stbenjam** [comment]
ACK

**Pull request merged**